web_accounts_list.json

```diff
@@ -2818,6 +2807,28 @@
          "known_accounts" : ["naughty_nymphomaniac", "hellfireclub", "fabswingers.com"],
          "category" : "dating",
          "valid" : true
+      },
+      {
+         "name" : "Magix",
+         "check_uri" : "https://www.magix.info/us/users/profile/{account}/",
+         "account_existence_code" : "200",
+         "account_existence_string" : "About me",
+         "account_missing_string" : "Page not found",
+         "account_missing_code" : "200",
+         "known_accounts" : ["baywolfmusic", "johnebaker"],
+         "category" : "music",
+         "valid" : true
+      },
+      {
+         "name" : "Tunefind",
+         "check_uri" : "https://www.tunefind.com/user/profile/{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : "Achievements",
+         "account_missing_string" : "Page not found",
+         "account_missing_code" : "200",

```

Comment:
After testing this, I think this should be a 404. The `python3 ./check_online_presence.py -s tunefind` comes back with a false negative for the random name. Can you check it out?

Reply:
My bad. I used `web_accounts_list_checker.py` to test with. :(

404 doesn't work, if it's an incorrect profile it 302's to a 404 so the 302 seems to work better. I've tested with `check_online_presence.py` now and that seems to go through without errors now. Will submit updated .json.

Reply:
Looks good. Thanks for fixing.